test/langtools/tools/javac/limits/ArrayDims2.java

```diff
@@ -35,6 +35,4 @@ class ArrayDims2 {
         [][][][][][][][][][]
```

Comment:
given that you are in the neighborhood :), the main method at: ArrayDims3.java can also be removed. Or can be done in another phase as you please

Reply:
Sure, no problem... and same thing for `ArrayDims1.java` it looks like.